### PR TITLE
Update some packages

### PR DESCRIPTION
**Pull request by krnowak**
The updated packages are mostly the "quick wins" ones. What's left is to update is web framework, database ORM and logging. The last two are kinda coupled since the ORM we use currently is using the current logger used in nebraska.

### CODE REVIEW

File: pkg/api/packages.go [hunk 1]
```diff
@@ -240,18 +239,22 @@ func (api *API) updatePackageBlacklistedChannels(tx *runner.Tx, pkg *Package) er
 		return err
 	}
 
-	newChannelsBlacklist := set.NewNonTS()
+	e := struct{}{}
```
Comment (joaquimrocha) [inline pkg/api/packages.go]: nitpick: I hope this is standard procedure in Go. As a non-expert in Go, I don't love the non-constant/single letter var just to assign as an empty value to use map as a set.
If the name/non-constness is biz as usual, carry on :smile_cat:

File: pkg/omaha/omaha.go [hunk 2]
```diff
@@ -6,7 +6,7 @@ import (
 	"io"
 	"strconv"
 
-	omahaSpec "github.com/aquam8/go-omaha/omaha"
+	omahaSpec "github.com/coreos/go-omaha/omaha"
```
Comment (joaquimrocha) [inline pkg/omaha/omaha.go]: I am very glad to seeing this change. Thanks!

File: pkg/omaha/omaha.go [hunk 3]
```diff
@@ -67,72 +67,64 @@ func (h *Handler) Handle(rawReq io.Reader, respWriter io.Writer, ip string) erro
...
 
-	return h.crAPI.RegisterEvent(machineID, appID, group, eventType, eventResult, event.PreviousVersion, event.ErrorCode)
+func (h *Handler) getStatusMessage(crErr error) omahaSpec.AppStatus {
+	return omahaSpec.AppStatus(h.getStatusMessageStr(crErr))
 }
 
-func (h *Handler) getStatusMessage(crErr error) string {
+// TODO(krnowak): This seems to return a bunch of custom errors. Not
+// sure if we should try to match it to the standard or extra
+// AppStatus constants.
```
Comment (joaquimrocha) [inline pkg/omaha/omaha.go]: Yeah, I'd try to match as much as possible to the standard ones.

File: pkg/api/packages.go (hunk 1 above)
Comment (krnowak) [reply pkg/api/packages.go]: First of all, there isn't anything like **standard** procedure or whatever in Go. There are **idiomatic** things. **Idiomatic**. I think I hate this word, by the way. ;)

So anyway, not sure if it is a standard thing. Probably not. Wanted to save some typing, but that's a silly reason. Will fix it.

File: pkg/omaha/omaha.go (hunk 3 above)
Comment (krnowak) [reply pkg/omaha/omaha.go]: Thing is that there isn't many of the standard constants:

https://github.com/coreos/go-omaha/blob/master/omaha/codes.go#L147-L159

So, for now I'll leave it as is and file an issue about it. I will need to check the omaha specification to see what app statuses they specify.

**Pull request merged**